packages/core/src/charts/wordcloud.ts

```diff
@@ -12,7 +12,7 @@ import { WordCloud } from '@/components/graphs/wordcloud'
 export class WordCloudChart extends Chart {
 	model = new WordCloudModel(this.services)
 
-	constructor(holder: HTMLDivElement, chartConfigs: ChartConfig<WorldCloudChartOptions>) {
+	constructor(holder: HTMLDivElement, chartConfigs: ChartConfig<WordCloudChartOptions>) {
```

Comment:
should we do `WordCloudChartOptions | WorldCloudChartOptions`? typescript could still throw an error not matching the type names

Reply:
Updated.